client/my-sites/earn/ads/wrapper.jsx

```diff
@@ -190,16 +190,23 @@ class AdsWrapper extends Component {
 	}
 
 	renderUpsell() {
-		const { translate } = this.props;
+		const { siteSlug, translate } = this.props;
+		const bannerURL = `/checkout/${ siteSlug }/premium`;
 		return (
-			<UpgradeNudgeExpanded
+			<UpsellNudge
+				callToAction={ translate( 'Upgrade' ) }
 				plan={ PLAN_PREMIUM }
 				title={ translate( 'Upgrade to the Premium plan and start earning' ) }
-				subtitle={ translate(
+				description={ translate(
 					"By upgrading to the Premium plan, you'll be able to monetize your site through the WordAds program."
 				) }
-				highlightedFeature={ FEATURE_WORDADS_INSTANT }
-				benefits={ [
+				feature={ FEATURE_WORDADS_INSTANT }
+				href={ bannerURL }
+				showIcon
+				event="calypso_upgrade_nudge_impression"
+				tracksImpressionName="calypso_upgrade_nudge_impression"
+				tracksClickName="calypso_upgrade_nudge_click"
```

Comment:
Noted this is a different event name than the one we use currently:

https://github.com/Automattic/wp-calypso/blob/7e5c83c7eb3c9da2a76f23e5745231e8fbb43ef7/client/blocks/upgrade-nudge-expanded/index.jsx#L47

Is this intended or should we keep using the same one? If the later, we'd need to also set a `tracksClickProperties` prop to keep tracking the same properties upon clicking on the CTA.

Reply:
Good catch, that's a typo. :D